src/App.css

```diff
@@ -1,38 +1,85 @@
-.App {
+.main-heading {
+  font-size: 7vw;
   text-align: center;
+  font-weight: 300;
+  background-color: rgb(80, 80, 80, 0.3);
+  padding: 5rem;
+  width: 80%;
+  margin: auto;
 }
```

Comment:
You do not have to use css like this for this project please remove them

Reply:
ok